alerting/src/test/kotlin/com/amazon/opendistroforelasticsearch/alerting/MonitorRunnerIT.kt

```diff
@@ -441,7 +446,11 @@ class MonitorRunnerIT : AlertingRestTestCase() {
         verifyAlert(alerts.single(), monitor, ACTIVE)
     }
 
-    fun `test execute monitor with bad search`() {
+    /*
+     * Commented. This test is not relevant anymore. We can't create a monitor with illegal chars.
+     * This is replaced with MonitorRestApiIT.`test creating a monitor with illegal index name`
+     */
+    /*fun `test execute monitor with bad search`() {
```

Comment:
Why note remove the code since it has essentially became dead code?

Reply:
Good point. Adding comments here and deleting the dead code.
```
     * Commented. This test is not relevant anymore. We can't create a monitor with illegal chars.
     * This is replaced with MonitorRestApiIT.`test creating a monitor with illegal index name`
```